SWLOR.Game.Server/Event/Conversation/HasCredits.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NWN;
+using SWLOR.Game.Server.Data.Contracts;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.Service.Contracts;
+using Object = NWN.Object;
+
+namespace SWLOR.Game.Server.Event.Conversation
+{
+    public class HasCredits : IRegisteredEvent
+    {
+        private readonly INWScript _;
+
+        public HasCredits(INWScript script)
+        {
+            _ = script;
+        }
+
+        public bool Run(params object[] args)
+        {
+                NWPlayer oPC = _.GetPCSpeaker();
```

Comment:
I'm going to be nitpicky here. Can you fix the spacing here? Looks like there's an extra set of tabs.